include/boost/beast/core/string.hpp

```diff
@@ -56,6 +66,8 @@ struct iequal
     {
         return iequals(lhs, rhs);
     }
+
+    struct is_transparent {};
```

Comment:
I would use `using is_transparent = void;` if possible, since it prevents doxygen from emitting a separate class type.

Reply:
I'll check the standard and make it so.